GliaWidgets/Glia.swift

```diff
@@ -321,3 +323,140 @@ public class Glia {
         )
     }
 }
+
+extension Glia {
+    // swiftlint:disable function_body_length
+        public func presentSurvey() {
+
+            let decoder = JSONDecoder.gliaApi
+
+            let utf8Data = """
```

Comment:
This is temporary example of survey specific JSON and it will be removed after proper integration, right?

Reply:
yes, it is just to present it quickly and check that this component works well.